src/data/community-repos.json

```diff
@@ -158,6 +158,13 @@
 		"authors": [{ "name": "jmorjsm", "url": "https://github.com/jmorjsm" }],
 		"has_variants": true
 	},
+	{
+		"name": "JetBrains IDE (IntelliJ, PyCharm, GoLand, etc.)",
```

Comment:
```suggestion
		"name": "JetBrains IDEs",
```

Reply:
Feel free to add "IntelliJ" etc. to the tags, as that should still be searchable. This name shows on the small-ish card, though so it would be too long there.